addons/event_booth_sale/static/src/js/sale_product_field.js

```diff
@@ -48,7 +48,7 @@ patch(SaleOrderLineProductField.prototype, {
             {
                 additionalContext: actionContext,
                 onClose: async (closeInfo) => {
-                    if (!closeInfo || closeInfo.special) {
+                    if (!closeInfo || closeInfo.special || closeInfo.dismiss) {
```

Comment:
This should be the safest, same below
```suggestion
                    if (!closeInfo.?eventBoothConfiguration || closeInfo.special || closeInfo.dismiss) {
```